cmd/sandbox/start.go

```diff
@@ -57,11 +59,16 @@ func startSandboxCluster(ctx context.Context, args []string, cmdCtx cmdCore.Comm
 
 func startSandbox(ctx context.Context, cli docker.Docker, reader io.Reader) (*bufio.Scanner, error) {
 	fmt.Printf("%v Bootstrapping a brand new flyte cluster... %v %v\n", emoji.FactoryWorker, emoji.Hammer, emoji.Wrench)
-	if err := docker.SetupFlyteDir(); err != nil {
+	if err := util.SetupFlyteDir(); err != nil {
 		return nil, err
 	}
 
-	if err := docker.GetFlyteSandboxConfig(); err != nil {
+	spec := util.ConfigTemplateSpec{
+		Host:     "localhost:30081",
+		Insecure: true,
+	}
+	configTemplate := util.ConfigTemplate + util.StorageTemplate
```

Comment:
move this to a function called createCombinedTemplate and return the concatenated templates.